mrp_repair_full_editable/README.rst

```diff
@@ -1,11 +1,18 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
 MRP Repair full editable
 ========================
 
 This module allows to add operations and fees in not invoiced or done repair
 operations.
 
+After confirm a repair order if the user change the invoice method,the workflow
+doesn't work properly. Therefor, the invoice method will be editable just in
+draft orders.
```

Comment:
After a repair order confirmation if a user changes the invoice method, the workflow won't work properly. Therefore, the invoice method has to be editable **only** in draft orders.